pkg/speccheck/builtin/process_deployment_test.go

```diff
@@ -87,67 +79,65 @@ func TestAddsHashToDeploymentSpecForEnvFrom(t *testing.T) {
 
 	spec := runtimeToUnstructured(t, &deploymentSpec)
 
-	rst := resourceSyncTask{
```

Comment:
We now don't need to construct a bogus `resourceSyncTask`. Just test stuff directly.